### DISCUSSION TITLE
Docs: unnecessary installation of history package in tutorial.md?

### DISCUSSION THREAD

**Discussion by azangru** [remix-run/react-router#8541] [General]
Hi! Sorry for posting this in discussions; I tried to open an issue first, but with the template system in place, it was ... difficult 🙂 

Anyway, it seems that there is a bit of a contradiction in the docs. The "Upgrading from v5" article [says](https://github.com/remix-run/react-router/blob/main/docs/upgrading/v5.md?plain=1#L150) the following:

```
You'll also want to remove the `history` dependency from your package.json. The `history` library is a direct dependency of v6 (not a peer dep), so you won't ever import or use it directly.
```

Whereas the Tutorial [asks](https://github.com/remix-run/react-router/blob/main/docs/getting-started/tutorial.md?plain=1#L56) the user to install the history package:

```
npm install react-router-dom@6 history@5
```

I suppose this means that the tutorial is out of date?

**Comment by timdorr** [comment]
Got rid of it in 1cf5f7e938ea4d5cff23ce88f9f952f0a343b7a0
